src/overlays/actors/ovl_En_Ta/z_en_ta.c

```diff
@@ -252,77 +299,78 @@ s32 func_80B142F4(EnTa* this, PlayState* play, u16 textId) {
 
     if ((ABS((s16)(this->actor.yawTowardsPlayer - this->actor.shape.rot.y)) <= 0x4300) &&
         (this->actor.xzDistToPlayer < 100.0f)) {
-        this->unk_2E0 |= 1;
+        this->stateFlags |= TALON_STATE_FLAG_TRACKING_PLAYER;
         func_8002F2CC(&this->actor, play, 100.0f);
     }
     return false;
 }
 
-void func_80B14398(EnTa* this, PlayState* play) {
+void EnTa_SleepTalkInKakariko(EnTa* this, PlayState* play) {
     if (Actor_TextboxIsClosing(&this->actor, play)) {
-        func_80B13AA0(this, func_80B14754, func_80B167FC);
+        EnTa_SetupAction(this, EnTa_IdleAsleepInKakariko, EnTa_AnimSleeping);
     }
 }
 
-void func_80B143D4(EnTa* this, PlayState* play) {
+void EnTa_SleepTalkInLonLonHouse(EnTa* this, PlayState* play) {
     if (Actor_TextboxIsClosing(&this->actor, play)) {
-        func_80B13AA0(this, func_80B146F8, func_80B167FC);
+        EnTa_SetupAction(this, EnTa_IdleAsleepInLonLonHouse, EnTa_AnimSleeping);
     }
 }
 
-void func_80B14410(EnTa* this) {
+void EnTa_SetWokenFlagsAndActionFunctions(EnTa* this) {
```

Comment:
maybe `SetupWokenUp`? would want this to be more about what it means for talons action as opposed to "setting flags and action function", if that makes sense

Reply:
think the "woken" functions can also just be phrased as "awake"
so, this function can be like `EnTa_SetupAwake` which will then pass you to `EnTa_IdleAwakeInCastle` or `EnTa_IdleAwakeInKakariko`, same goes for the other "woken" functions. Think it just makes more sense as "talon is currently awake" rather than "talon has been awoken"